privacy_policy/en/privacy-notice-website.md

```diff
@@ -0,0 +1,71 @@
+You are not required to disclose any personal information in order to download any of our products.
+ 
+### When you purchase Adguard products
+When you pay for Adguard products the billing is generally handled by a third-party service provider. In most cases the service provider is acting as our reseller; thus, you will be making your purchase from the service provider directly, and not from Adguard. The handling of your personal information will be governed by any privacy policy or terms of service published by the service provider. (LEARN MORE)
+
+We save the information you enter on the order page. This information can be used in the future if you request your license key to be recovered.
+
+The third-party service provider may transmit some of your personal information to us, but not including payment information, unless you specifically agree (as may be the case in the event of a purchase with auto-renewal). We may use the information we receive (not including payment information) to verify your registration or license status, to contact you about the status of your account, or for renewal of your subscription, if applicable.  (/LEARN MORE)
+
+### When you leave a comment
+We use Disqus for comment sections on reports.adguard.com and blog.adguard.com. In this case we do not collect any data about you, but please note that data might be collected by Disqus and managed according to their [privacy policy](https://help.disqus.com/customer/portal/articles/466259-privacy-policy). 
+
+### When you use Adguard Account
+Adguard Account (https://adguard.com/login.html) is a tool which permits you to register multiple license using a single registration and authentication system. If you choose to use Adguard Account you will be asked to provide your email address. (LEARN MORE) 
+You will also be asked to select a password. We may use this information for purposes of validating your license, providing support, delivering additional products and services, and enabling you to change your Adguard Account and/or product settings. (/LEARN MORE)
+
+### Third-party content and sites
+If we offer links to third-party sites such as Twitter, Facebook, or others - they do not share any data unless you specifically click the button.
+
+### News subscriptions
+Our website offers news and information by subscription including newsletters, blogs, or others. If you decide to subscribe to these services you may be asked to provide your email address. 
+
+Nevertheless you can always unsubscribe from our newsletters them via Adguard account or by clicking “Unsubscribe” button in the newsletter itself.  
+
+### Support Center
+You may join the Support Center at kb.adguard.com. Certain features require registration in order to participate. When you register for the Support Center you will be asked to provide your full name and email address, and select a password. No other information is required to register. This information is not shared with other users.
```

Comment:
That's redundant, you cannot join anymore, KB is just a read-only knowledge base